Src/EventCounterCollector/EventCounterCollector/EventCounterListener.cs

```diff
@@ -156,22 +156,24 @@ private void ExtractAndPostMetric(string eventSourceName, IDictionary<string, ob
                 double actualValue = 0.0;
                 double actualInterval = 0.0;
                 int actualCount = 0;
+                string counterName = string.Empty;
+                string counterDisplayName = string.Empty;
                 foreach (KeyValuePair<string, object> payload in eventPayload)
                 {
                     var key = payload.Key;
                     if (key.Equals("Name", StringComparison.OrdinalIgnoreCase))
                     {
-                        var counterName = payload.Value.ToString();
-                        if (this.countersToCollect[eventSourceName].Contains(counterName))
-                        {
-                            metricTelemetry.Name = eventSourceName + "|" + counterName;
-                        }
-                        else
+                        counterName = payload.Value.ToString();
+                        if (!this.countersToCollect[eventSourceName].Contains(counterName))                        
                         {
                             EventCounterCollectorEventSource.Log.IgnoreEventWrittenAsCounterNotInConfiguredList(eventSourceName, counterName);
                             return;
                         }
                     }
+                    else if (key.Equals("DisplayName ", StringComparison.OrdinalIgnoreCase))
```

Comment:
Is the space after DisplayName intentional?

Reply:
good catch.! our automated tests wouldn't have caught that as we need 3.0 infra to do the testing.